<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>José Manuel Díez Botella</author>
 <class>introfecha</class>
 <widget class="QDialog" name="introfecha">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fecha operación</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>FECHA:</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="fechadateEdit">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="lineWidth">
        <number>4</number>
       </property>
       <property name="midLineWidth">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>DIARIO:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="diariocomboBox">
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="nocerrarcheckBox">
     <property name="text">
      <string>No cerrar ventana - informar asiento y/o secuencia</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="acpetarpushButton">
       <property name="text">
        <string>&amp;Tabla</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/Files-New-File-icon.png</normaloff>:/iconos/graf/Files-New-File-icon.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelarpushButton">
       <property name="text">
        <string>&amp;Salir</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/borrar.png</normaloff>:/iconos/graf/borrar.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="herramientas_groupBox">
     <property name="title">
      <string>Herramientas</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="varios_sop_pushButton">
        <property name="text">
         <string>Varios tipos soportado</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="aib_pushButton">
        <property name="text">
         <string>AIB - Inversión sujeto pasivo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="varios_rep_pushButton">
        <property name="text">
         <string>Varios tipos repercutido</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="eib_pushButton">
        <property name="text">
         <string>EIB - Prestación servicios UE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="predef_pushButton">
        <property name="text">
         <string>Asientos predefinidos</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fechadateEdit</tabstop>
  <tabstop>diariocomboBox</tabstop>
  <tabstop>nocerrarcheckBox</tabstop>
  <tabstop>acpetarpushButton</tabstop>
  <tabstop>cancelarpushButton</tabstop>
  <tabstop>varios_sop_pushButton</tabstop>
  <tabstop>aib_pushButton</tabstop>
  <tabstop>varios_rep_pushButton</tabstop>
  <tabstop>eib_pushButton</tabstop>
  <tabstop>predef_pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>acpetarpushButton</sender>
   <signal>clicked()</signal>
   <receiver>introfecha</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelarpushButton</sender>
   <signal>clicked()</signal>
   <receiver>introfecha</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
